File: lodge-spot-go-be/src/AccommodationManagement/JetSetGo.AccommodationManagement.Domain/Accommodation/Accommodation.cs
using JetSetGo.AccommodationManagement.Domain.Accommodation.Enum;
using JetSetGo.AccommodationManagement.Domain.Accommodation.ValueObjects;

namespace JetSetGo.AccommodationManagement.Domain.Accommodation;

public class Accommodation
{
    public Guid Id { get; set; }
    public string Name { get; set; } = null!;
    public Address Address { get; set; } = null!;
    public List<string> Amenities { get; set; } = null!;
    public List<AccommodationPhoto> Photos { get; set; } = null!;
    public int MaxGuests { get; set; }
    public int MinGuests { get; set; }
    
    public List<SpecalPrice> SpecalPrices { get; set; }
}

File: lodge-spot-go-fe/src/app/common/model/accommodation.ts
import {Address} from "./addres";
import {SpecialPrice} from "./specialPrice";
import {Photo} from "./Photo";

export interface Accommodation {
  name?: string;
  address?:Address;
  max_guests?:number;
  min_guests?:number;
  amenities?:string[];
  photos?:string[];
  specialPrices?:SpecialPrice[]
}
